hw3/03-react/game-of-thrones/src/views/Houses.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { Component } from "react";
+import axios from "axios";
+import { Doughnut } from "react-chartjs-2";
+import { Chart as ChartJS, ArcElement, Tooltip, Legend } from "chart.js";
+ChartJS.register(ArcElement, Tooltip, Legend);
+
+class Houses extends Component {
+  state = {
+    charactersCount: [],
+    houses: [],
+  };
+  componentDidMount() {
+    axios.get(`https://thronesapi.com/api/v2/Characters`).then((res) => {
```

Comment:
Knowledge of how to work with classes is important, but these days it is more common to use functional components. Is there a specific reason a class is used here?